src/js/quizapiWrapper.js

```diff
@@ -0,0 +1,69 @@
+import fetch from "node-fetch";
+export class QuizapiWrapper {
+    constructor(category) {
+        this.apiKey = "EAco9tkerMBqitin1irWhCDipOxYHxJ5A5xe1nqQ"
+        this.data;
```

Comment:
This expression does nothing. It still be undefined as it was. You can assign `null` to it to indicate the absence of any value, an empty array or remove it.